Sources/Playbook/Components/Time/TimeCatalog.swift

```diff
@@ -0,0 +1,182 @@
+//
+//  Playbook Swift Design System
+//
+//  Copyright © 2024 Power Home Remodeling Group
+//  This software is distributed under the ISC License
+//
+//  TimeCatalog.swift
+//
+
+import SwiftUI
+
+struct TimeCatalog: View {
+  var body: some View {
+    ScrollView {
+      VStack(spacing: Spacing.medium) {
+        PBDoc(title: "Default") {
+          defaultView
+        }
+        PBDoc(title: "Sizes") {
+          sizesView
+        }
+        PBDoc(title: "Time Zones") {
+          handlingTimeZones
+        }
+        PBDoc(title: "Alignment") {
+          alignmentView
+        }
+        PBDoc(title: "Unstyled") {
+          unstyledView
+        }
+      }
+      .padding(Spacing.medium)
+    }
+    .background(Color.background(Color.BackgroundColor.light))
```

Comment:
```suggestion
    .background(Color.background(.light))
```

Reply:
For timeIcon (the clock icon) I need to differentiate between the clock,  the time and the time zone being bold or not. There are instances where the need to operate separately. I did add the textcolor computed property and added it to the time var!